Pipelines/Templates/update-deployment-settings.yml

```diff
@@ -249,6 +261,38 @@ steps:
             Authorization = "Bearer $env:SYSTEM_ACCESSTOKEN"
         }
         $buildDefinitionResponseResults = $fullBuildDefinitionResponse.value
+        if($buildDefinitionResponseResults.length -ne $deploymentSteps.Count) {
+            New-Item -ItemType "directory" -Name "../coe-starter-kit-source"
+            cd ../coe-starter-kit-source
+            git clone -b "mochs-aa4pp-1562" "https://github.com/microsoft/coe-starter-kit.git"
```

Comment:
will we be keeping this branch? Or, is this to be used for testing and will be changed later?

Reply:
I need to update that to main once that's merged. Thanks for pointing it out.